.travis.yml

```diff
@@ -1,59 +1,82 @@
-os: linux
-dist: xenial
+cache: npm
+
+os:
+  - osx
+  - linux
+
+osx_image:
+  - xcode11
+
+dist: bionic
 
 language: node_js
+
 node_js:
-  - "10"
-cache: npm
-services:
-  - docker
+  - lts/dubnium
+  - lts/erbium
+  - 12.13.0
 
 addons:
   sonarcloud:
     organization: "sakuli"
     token:
       secure: "kyPFHtVbo9gHNp7fy2B3d2I6g4/+v4h1/rjLm3LnLx1XECI13F/htxLI6tWOQGNnWeC3MRAMsrd2BJ2/ovJY0qz3EtfMBuFByEbwPiLobB+YJqaTTTnXcNsPVN+AibkbGk9jQEiO/2TIebo+29XtBH6EQLdsew1qNxGAXPCHyCFtrJT1x9jj+RO8RFxTP3PMFKzZaXAGd7Twl4AcWPTd6EYUCnV+2rq/mkIqmZwH0ckYIiVnIKm5g69P3i+LLabUl/tdftXU4+J+BHaHxwb7X+D0k9NsTXPWn19grmC8FdubO7GNX5zf+gvHX0d40oZGGYYe/55CSJKS0Coerj9Wmd380ZGzZZwnZl9pfyh6BITOWJlOUw8WoILX2w8k0jiZVF2puQ/h82+m9jfnPuk9v6cnIsJmO/2cSDz3T0ZznbhA4W/yGdsEfK0SKVn3VjyV6Gbd2R7YvZC3hag5UgRz2L8odIXn7y3U9KXQhpquxX/HLkVOKd2w/PXCcOX1BsMUcgjQiP2VfZuZKkGvkFg+E/iLovoSwlG0FYRu2Qmu5C74saz1K4H5IyVe3WzZZzWoLWSWwdWN0ZEXmpFI5Lxyy/3H6x4b+mkVld7j1ProuxQdf1TjFoYd0OWYcxVcedG6FdaNDmGc3FMtmVuSjeBYcpbMyU7d7l7xdigeNTK86J8="
+  homebrew:
+    casks:
+      - google-chrome
+    update: true
 
-before_install:
-  - sudo apt-get install -y build-essential libxt-dev libxtst-dev libpng++-dev libxinerama-dev
-  - docker pull selenium/standalone-chrome-debug
-  - docker pull selenium/standalone-firefox-debug
-  - docker pull s1hofmann/nut-ci
-  - docker run -it -d --name nut-ci --user $(id -u):$(id -g) -v ${PWD}:${PWD}:rw s1hofmann/nut-ci:latest bash
-  - docker run -d -p 4444:4444 selenium/standalone-firefox-debug
-  - docker run -d -p 4445:4444 selenium/standalone-chrome-debug
-  - npm i -g lerna gh-pages
+services:
+  - docker
+  - xvfb
 
+before_install:
+  - "bash ./.build/setup_${TRAVIS_OS_NAME}.sh"
 install: true
-
 script:
-  - travis_wait npm ci
-  - lerna run build > /dev/null 2>&1
-  - FIREFOX_WD_URL=http://localhost:4444/wd/hub CHROME_WD_URL=http://localhost:4445/wd/hub npm test -- --runInBand --ci --bail
-  - sonar-scanner
```

Comment:
I just noticed that sonar cloud is not reporting back. Is it because this is missing in e.g. `run_linux.sh`?